srcpkgs/pinebookpro-uboot/files/kernel.d/uboot

```diff
@@ -1,20 +1,27 @@
 #!/bin/sh
 
 kver=${2}
-uuid=$(cat /etc/fstab | awk '$2 == "/" { print $1 }' | sed 's/^UUID=//')
+uuid=$(cat /etc/fstab | sed -r '/^\s*#/d' | awk '$2 == "/" { print $1 }' | sed 's/^UUID=//')
 dev=$(blkid -U "${uuid}")
 partuuid=$(blkid -o value -s PARTUUID "${dev}")
 bootpart=$(df -P /boot | tail -1 | awk '{ print $6 }')
 bootstrip() {
     echo ${1} | sed "s,^${bootpart}/,/,"
 }
+root="PARTUUID=$partuuid"
+
+[ -z "$partuuid" ] && {
+  # UUID search failed, trying fstab directly
+  root=$(cat /etc/fstab | sed -r '/^\s*#/d;s/UUID=/PARTUUID=/' | awk '$2 == "/" {print $1}')
```

Comment:
I personally don't care about `UUID=` or `PARTUUID=` and I'm using neither of them in my `/etc/fstab`. My intention is to make this script work the same as the original which apparently only supports `UUID=` in the fstab plus also support fstab with no `*UUID=`. Personally, I'm using encrypted root as `/dev/mapper/croot`. 

Therefore, if you think, we only should support `PARTUUID=` because `UUID=` is wrong, I don't object.

Reply:
I'm referring to the current version of this package (version 2022.04, revision 2). Sorry if I didn't express myself clearly enough.